storage/entry_query_builder.go

```diff
@@ -32,7 +32,9 @@ func (e *EntryQueryBuilder) WithSearchQuery(query string) *EntryQueryBuilder {
 		nArgs := len(e.args) + 1
 		e.conditions = append(e.conditions, fmt.Sprintf("e.document_vectors @@ plainto_tsquery($%d)", nArgs))
 		e.args = append(e.args, query)
-		e.WithOrder(fmt.Sprintf("ts_rank(document_vectors, plainto_tsquery($%d))", nArgs))
+
+		// 0.0000001 = 0.1 / (seconds_in_a_day)
+		e.WithOrder(fmt.Sprintf("ts_rank(document_vectors, plainto_tsquery($%d)) - extract (epoch from now() - published_at)::float * 0.0000001", nArgs))
```

Comment:
Can you explain how this works?

Reply:
ts_rank value has range 0.0 - 1.0. So If entry is relevant, but age is 1 day, so it's results search rank will be 0.9. If multiple entries have close relevance score according to ts_rank, than this "age delta" would prioritise more recent results.